core-bundle/contao/themes/flexible/styles/components/input-map.pcss

```diff
@@ -3,6 +3,10 @@ ul.sortable li,
 [data-contao--input-map-target="source"] {
     position: relative;
     cursor: move;
+
+    &:has(.gimage) {
+        display: inline-flex;
```

Comment:
I think if we add this here, we can remove the old format definition, right?

<img width="268" height="188" alt="" src="https://github.com/user-attachments/assets/88b6c151-9713-49f6-8178-a1fb26efafb1" />

Reply:
The old format comes just above it if you look closer.

Reply:
See 92ccc0548472c02fc4dd4ba4f3fdad8de408b4ff, the styles are still needed as there is literally a ToDo to remove it when MooSortable is gone, however there was no need to have it nested inside the ul.sortable as well.